scripts/android/autopower/autopower.py

```diff
@@ -23,20 +24,19 @@
 import sys
 import time
 
-simulation = False
 
-durationSecond = 1200
+# Parameters to change
+durationSecond = 300 # At least 40 for accuracy
 burstSize = 64
```

Comment:
Were the tests run with 64 or 96?

Reply:
I thought I had changed it, but I must have rewrote when rolling some changes back. The current tests were taking with 64 bursts.

Reply:
OK, just make a note in the results log.